crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -508,6 +508,19 @@ impl<'w> EntityMut<'w> {
         self.world
     }
 
+    /// Access the inner `&mut World` within `use_world` and run [`Self::update_location`].
```

Comment:
I've read the entire comment for this method but I have no idea what it means. This sounds like a fairly advanced usage. I don't know what `use_world()` does, nor `update_location()`. Can we try to give a little bit more detail, and maybe give an example of when this method is useful?

Reply:
`use_world` is the function parameter. `update_location` is linked in the doc, because it is surrounded in [], if someone is reading the documentation and doesn't know what `update_location` is, they should be able to click on the link and read the `update_location` documentation.

Reply:
```suggestion
    /// Access the inner `&mut World` within the `use_world` function argument and run [`Self::update_location`].
```